src/components/CreateAnalysis/ModelSelection.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { SyntheticEvent, useEffect, useState } from 'react'
+import plugins from '../../api/app.config'
+import {
+    Dropdown,
+    DropdownToggle,
+    DropdownItem
+  } from '@patternfly/react-core';
+  import CaretDownIcon from '@patternfly/react-icons/dist/js/icons/caret-down-icon';
+
+interface ModelSelectionProps {
+    useXray: boolean,
```

Comment:
Suggestion for future: try to avoid naming variables (especially boolean variables) as `useXXXXXXXX`. Reserve `useXXXXX` for hooks